isis/src/base/objs/OriginalLabel/OriginalLabel.cpp

```diff
@@ -76,7 +108,7 @@ namespace Isis {
   Pvl OriginalLabel::ReturnLabels() {
     Pvl pvl;
     stringstream os;
-    for(int i = 0; i < p_nbytes; i++) os << p_buffer[i];
+    os << m_originalLabel;
     os >> pvl;
     return pvl;
```

Comment:
This should just need to return m_originalLabel

Reply:
Yeahhhhhh this was the result of trying to fix a bug by changing the wrong part of the code, then I got it working and didn't change everything back.

Good catch